packages/ui/src/Textarea/index.tsx

```diff
@@ -7,20 +7,34 @@ import { Text } from '../Text';
 type TextareaProps = {
   label?: string;
   width?: string;
+  height?: string;
 } & TextareaHTMLAttributes<HTMLTextAreaElement>;
 
 export const Textarea = forwardRef(function Textarea(
-  { label, width = '100%', onChange, ...props }: TextareaProps,
+  { label, width = '100%', height = '150px', onChange, ...props }: TextareaProps,
   ref: ForwardedRef<HTMLTextAreaElement>
 ) {
+  const handleInput = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
```

Comment:
```suggestion
  const handleInput = (e: ChangeEvent<HTMLTextAreaElement>) => {
```

컨벤션상 React.은 빼는게 좋을것같아용

Reply:
넵